tests/platform/cloud/AcceptanceTests_untrustedSettingsGet.js

```diff
@@ -230,126 +230,6 @@ gpii.tests.cloud.oauth2.untrustedSettingsGet.disruptedTests = [
 
```

Comment:
This test has the name "A successful workflow for no authorized solutions". I think that we can rename it to "A successful workflow for retrieving settings" as we don't have authorized solutions anymore.